addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyMissingScanRule.java

```diff
@@ -111,6 +109,19 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                     .setWascId(15) // WASC-15: Application Misconfiguration
                     .raise();
         }
+        // ... or as a warning at LOW thresshold if one of the obsolete headers is found
+        if (AlertThreshold.LOW.equals(this.getAlertThreshold())
+                && (xCspHeaderFound || xWebKitHeaderFound)) {
+            newAlert()
```

Comment:
IMO it should use other name, e.g. `Obsolete Content Security Policy (CSP) Header`

Reply:
I think he was suggesting a name for the Alert in this case.

Reply:
I have made the two more distinct